### PR TITLE
MODINVSTOR-315 -Institution, campus and library code should be required.

### PR DESCRIPTION
**Pull request by VRohach**
raml-files changed.

### CODE REVIEW

**Comment by VRohach** [comment]
> Please bump up the version of the location-units interface.

Sure. Done

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -225,7 +225,7 @@
     },
     {
       "id": "location-units",
-      "version": "1.1",
+      "version": "2.0",
```
Comment (marcjohnson-kint) [inline descriptors/ModuleDescriptor-template.json]: This version change should also be made in the RAML file

Comment (VRohach) [reply descriptors/ModuleDescriptor-template.json]: @marcjohnson-kint  Thank you! I changed this version to 2.0 in the "locationunit.raml"-file.

**Comment by VRohach** [comment]
This PR requires PRs for other modules: 

1. https://github.com/folio-org/mod-circulation/pull/360
2. https://github.com/folio-org/ui-tenant-settings/pull/55
3. https://github.com/folio-org/ui-circulation/pull/438
4. https://github.com/folio-org/ui-inventory/pull/716

**Comment by marcjohnson-kint** [comment]
> This PR requires PRs for other modules:
> 
> 1. [folio-org/mod-circulation#360](https://github.com/folio-org/mod-circulation/pull/360)
> 2. [folio-org/ui-tenant-settings#55](https://github.com/folio-org/ui-tenant-settings/pull/55)
> 3. [folio-org/ui-circulation#438](https://github.com/folio-org/ui-circulation/pull/438)
> 4. [folio-org/ui-inventory#716](https://github.com/folio-org/ui-inventory/pull/716)

Once those pull requests are approved, feel free to remove the do not merge label from this change

**Comment by mrdpopov** [comment]
@marcjohnson-kint It has been decided postpone adding further constraints to the codes now as there are separate stories for that.

**Comment by mrdpopov** [comment]
@VRohach are there tests that create insitutions, libraries and campuses without code?

**Comment by VRohach** [comment]
> @VRohach are there tests that create insitutions, libraries and campuses without code?

There were tests without code before my changes in this PR, but I added them regarding the implementation of this story.

**Comment by VRohach** [comment]
> Please add tests that create libraries/insitution/campuses without codes.

Done. Could you please review it?

**Pull request merged**